src/Containers/Invite/Invite.tsx

```diff
@@ -22,149 +22,169 @@ export interface InviteProps {
         placeholder: string;
         type: string;
         subLabel?: string;
-    }[]; 
+    }[];
 }
 
 interface InputFieldsProps {
     label: string;
-    subLabel?: string
+    subLabel?: string;
     name: string;
     type: string;
     placeholder: string;
     as?: string;
 }
 
 /**
- * @param  
+ * @param
  * @returns label, sublabel, input props & error message
  */
- const IField = ({ label, subLabel, ...rest }: InputFieldsProps) => { 
+const IField = ({ label, subLabel, ...rest }: InputFieldsProps) => {
     const [field, meta] = useField(rest);
     return (
-    <>
-        {meta.touched && meta.error ? <SmallText color="primary">{meta.error}</SmallText> : null}
-        <FormGroup>
-            <Label>
-                {label}
-                <SubLabel>{subLabel}</SubLabel> 
-            </Label>
-            <IDiv>
-                <Input {...field} {...rest} />
-            </IDiv>
-        </FormGroup>
-    </>
+        <>
+            {meta.touched && meta.error ? (
```

Comment:
Just use && and remove the ? : code

Make sure that it is booleans on the left side